DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -24,6 +24,27 @@ namespace ticl {
     // the first element is on the inner layer and
     // the outer element is on the outer layer.
     std::vector<std::array<unsigned int, 2> > edges;
+
+    // regressed energy
+    float regressed_energy;
+
+    // trackster ID probabilities
+    std::array<float, 7> id_probabilities;
```

Comment:
I would still prefer to have these probabilities use standard enums. The ParticleType from PF has an "undefined" category, though not an "ambiguous" category: https://github.com/cms-sw/cmssw/blob/master/DataFormats/ParticleFlowCandidate/interface/PFCandidate.h#L43-L52